gemini/evaluation/migration_guide_preview_to_GA_sdk.ipynb

```diff
@@ -0,0 +1,2812 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "OzHQmCjBjZOs"
+   },
+   "outputs": [],
+   "source": [
+    "# Copyright 2024 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5hCmIhlJjZOt"
+   },
+   "source": [
+    "# Gen AI Evaluation Service SDK Preview-to-GA Migration Guide\n",
```

Comment:
Can you add more information about why and when this migration needs to happen?

Reply:
The GA SDK is already released. This migration guide is for current Public Preview SDK users to fix their breaking changes when moving to GA version SDK.